hack/verify-diff.sh

```diff
@@ -1,10 +1,21 @@
 #!/bin/bash
-FILE_DIFF=$(git ls-files -o --exclude-standard)
 
-if [ "$FILE_DIFF" != "" ]; then
-  echo "Found untracked files:"
-  echo "$FILE_DIFF"
-  exit 1
+status=0
+# Checks that there are no uncommitted changes.
+if [[ -n "$(git status --porcelain .)" ]]; then
+    echo "[!] Uncommitted files. Commit changes and run 'make test'."
+    echo "$(git status --porcelain .)"
+    let status=1
 fi
 
-git diff --exit-code
+# Checks that changes under config directory are reflected in the bundle directory.
+# This is to ensure that the bundle is up-to-date with the latest changes.
+# If this fails, run 'make bundle' to update the bundle.
+if [[ -n "$(git status --porcelain config/)" ]] && [[ -z "$(git status --porcelain bundle/)"  ]]; then
+    echo ""
+    echo "[!] Config folder has changes, but Bundle does not. Check if Bundle is out-of-date."
+    echo "$(git status --porcelain config/)"
```

Comment:
Maybe also printing with  `echo "$(git status --porcelain bundle/)"`

Reply:
I don't understand, in this case bundle folder is not expected to have any change to show (unless the check above is wrong somehow)

Reply:
You are right, I was confused and I didn't see the `-z` beforehand..